python/spglib/spglib.py

```diff
@@ -35,8 +35,10 @@
 
 from __future__ import annotations
 
+import dataclasses
 import warnings
-from typing import TYPE_CHECKING
+from collections.abc import Iterator, Mapping
+from typing import TYPE_CHECKING, Any
```

Comment:
`Any` should be in `if TYPE_CHECKING`. `Iterator` as well

Reply:
`Any` is required for `Mapping[str, Any]`

Reply:
I thought `__future__.annotations` takes care of that. Maybe with `"Any"`?

Reply:
Now it works, but I don't follow why `"Any"` works. Do you have some reference?

Reply:
I find it https://peps.python.org/pep-0563/